### DISCUSSION TITLE
Help with garungorp mouse injection tutorial

### DISCUSSION THREAD

**Discussion by PinoIsReading** [garungorp/MouseInjectorDolphinDuck#13] [General]
I tried to follow the tutorial and use the same game used as example (red faction II), but i only get 2 or 3 addresses in cheat engine when searching for the game serial (SLUS_204.42) and none end with 3390 or 708 like he said in the video... i need help because i don't know how to continue, thanks.

tutorial: https://www.youtube.com/watch?v=LLtaIrMyQBc

**Comment by PinoIsReading** [comment]
i think i give up

**Reply by GoldenLumia** [reply]
This got me as well working with Duckstation. If you haven't, enable MEM_MAPPED inside Cheat Engine settings, that fixed it for me.

**Reply by PinoIsReading** [reply]
thank you i will try it some day

**Reply by McGeezo** [reply]
> This got me as well working with Duckstation. If you haven't, enable MEM_MAPPED inside Cheat Engine settings, that fixed it for me.

Have you by any chance worked on Syphon Filter? I wanna do it but have no idea what I'm doing since the tutorial is only for PCSX2 and I got lost at the debug/memory part

**Reply by GoldenLumia** [reply]
> > This got me as well working with Duckstation. If you haven't, enable MEM_MAPPED inside Cheat Engine settings, that fixed it for me.
> 
> Have you by any chance worked on Syphon Filter? I wanna do it but have no idea what I'm doing since the tutorial is only for PCSX2 and I got lost at the debug/memory part

I have not, apologies. However, I have worked on [multiple PS1 armored core games](https://github.com/GoldenLumia/MouseInjectorDolphinDuck), so I can tell you that most of the tutorial is transferable. The only hang-up is that PCSX2's debugger seems to be better than Duckstation, but Cheat Engine will get you most of the way there.
